test/extended/openstack/volumes.go

```diff
@@ -241,6 +241,78 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenStack pla
 			o.Expect(err).NotTo(o.HaveOccurred())
 			o.Expect(out).To(o.Equal(fileContent))
 		})
+
+		g.It("should create a cinder volume when using cinder default storage class", func(ctx g.SpecContext) {
+
+			var err error
+
+			cinderSc := FindDefaultStorageClassByProvider(oc, "cinder.csi.openstack.org")
+			o.Expect(cinderSc).NotTo(o.BeNil(), "default cinder-csi storageClass not found.")
```

Comment:
So we're saying that we can *always* expect Cinder CSI driver to be deployed, yes?

Reply:
yes! afaik, shiftstack installation should create this SC with the default annotation pointing to cinder-csi provisioner.

Reply:
I refactored a bit the FindStorageClassByProvider function because I noticed that manila SC does not have the "is_default" annotation.